src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -63,50 +59,97 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
                     {
-                        case ParameterLocation.Path:
-                            path = path.Replace("{" + param.Name + "}", HttpUtility.UrlEncode(valueStr));
-                            break;
-
-                        case ParameterLocation.Query:
-                            query.Append((query.Length == 0) ? "?" : "&");
-                            query.Append(param.Name);
-                            query.Append('=');
-                            query.Append(HttpUtility.UrlEncode(valueStr));
-                            break;
-
-                        case ParameterLocation.Header:
-                            headers.Add(param.Name, valueStr);
-                            break;
-
-                        default:
-                            throw new NotImplementedException($"{param.In}");
+                        // No schema, we send the raw value
+                        body = paramValue.ToObject().ToString();
                     }
+                    else
+                    {
+                        switch (param.In.Value)
+                        {
+                            case ParameterLocation.Path:
+                                path = path.Replace("{" + param.Name + "}", HttpUtility.UrlEncode(valueStr));
+                                break;
+
+                            case ParameterLocation.Query:
+                                query.Append((query.Length == 0) ? "?" : "&");
+                                query.Append(param.Name);
+                                query.Append('=');
+                                query.Append(HttpUtility.UrlEncode(valueStr));
+                                break;
+
+                            case ParameterLocation.Header:
+                                headers.Add(param.Name, valueStr);
+                                break;                           
+
+                            case ParameterLocation.Cookie:
+                            default:
+                                throw new NotImplementedException($"{param.In}");
+                        }
+                    }                    
                 }
             }
 
             var url = path + query.ToString();
 
             // $$$ Not handling Body yet...
-            var request = new HttpRequestMessage(_method, url);
+            var request = new HttpRequestMessage(_method, url);            
 
             foreach (var kv in headers)
             {
                 request.Headers.Add(kv.Key, kv.Value);
             }
 
+            if (!string.IsNullOrEmpty(body))
+            {                
+                // $$$ Might need System.Net.Http.FormUrlEncodedContent()
+                request.Content = new StringContent(body, Encoding.Default, _argMapper.ContentType);                
+            }
+
             return request;
         }
 
+        private string GetBody(OpenApiParameter param, Dictionary<string, FormulaValue> map)
```

Comment:
>string [](http://example.com/codeflow?start=16&length=6)

Should this just return the HttpContent directly? (that may scale as we move to non-string bodies)

Reply:
why ByteArrayContent instead of HttpContent?